### PR TITLE
D8CORE-5096: visually hid external link svgs in localist embeddable.

### PR DESCRIPTION
**Pull request by imonroe**
# READY FOR REVIEW

# Summary
- This is a hotfix (merge to master) which visually hides the external link SVG arrows within Localist embeddable widgets.  It addresses the visual bug in D8CORE-5096

# Review By (Date)
- asap

# Criticality
- Medium

# Urgency
- medium

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Create a Localist widget
3. verify the external link icons no longer appear in the embedded content.

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory? NO

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: modules/stanford_profile_styles/lib/scss/base/stanford_profile_styles.scss [hunk 1]
```diff
@@ -50,6 +50,27 @@ svg {
...
+  a {
+    &.su-link--external {
+      &::after {
+        @include hide-visually;
+      }
+    }
+  }
+}
+
+#lclst_widget_footer {
```
Comment (pookmish) [inline modules/stanford_profile_styles/lib/scss/base/stanford_profile_styles.scss:64]: What happens when you have 2 widgets on the same page?

Comment (imonroe) [reply modules/stanford_profile_styles/lib/scss/base/stanford_profile_styles.scss:64]: The id isn't going to change; that's coming from inside the embed content, even if there's more than one.  It's bad code from localist.

Comment (imonroe) [reply modules/stanford_profile_styles/lib/scss/base/stanford_profile_styles.scss:64]: Yeah, I just validated that by placing two widgets on the same page, and it works as expected.

image

Comment (jenbreese) [reply modules/stanford_profile_styles/lib/scss/base/stanford_profile_styles.scss:64]: I tried it in VoiceOver and the link was read as an External Link. I remember @cjwest  was concerned about that.

Comment (pookmish) [reply modules/stanford_profile_styles/lib/scss/base/stanford_profile_styles.scss:64]: Ok thanks for checking. That is bad code from Localist but what  can ya do.

**Review by pookmish — APPROVED** [review]

**Comment by cjwest** [comment]
@Jennifer R ***@***.***>,
If it is truly an external link, then it's okay.
________________________________

**Pull request merged**